testing/run_unit_tests.sh

```diff
@@ -6,13 +6,15 @@ TESTDIR=`dirname $SCRIPTPATH`
 BASEDIR=`readlink -f $TESTDIR/..`
 cd $BASEDIR
 
-$BASEDIR/venv/bin/coverage erase
+source venv/bin/activate
 
-PYTHONPATH=$BASEDIR/daq:$BASEDIR/mininet:$BASEDIR/faucet:$BASEDIR/forch $BASEDIR/venv/bin/coverage run \
+coverage erase
+
+PYTHONPATH=$BASEDIR/daq:$BASEDIR/mininet:$BASEDIR/faucet:$BASEDIR/forch coverage run \
```

Comment:
set separately and export -- this is getting ugly as a prefix

Reply:
Done